adapters/openrtb_util.go

```diff
@@ -52,21 +52,23 @@ func makeVideo(unit pbs.PBSAdUnit) *openrtb.Video {
 	}
 	mimes := make([]string, len(unit.Video.Mimes))
 	copy(mimes, unit.Video.Mimes)
-	pbm := make([]int8, 1)
+	pbm := make([]openrtb.PlaybackMethod, 1)
 	//this will become int8 soon, so we only care about the first index in the array
-	pbm[0] = unit.Video.PlaybackMethod
-	if pbm[0] == 0 {
-		pbm = nil
+	pbm[0] = openrtb.PlaybackMethod(unit.Video.PlaybackMethod)
+
+	protocols := make([]openrtb.Protocol, len(unit.Video.Protocols))
+	for _, protocol := range unit.Video.Protocols {
+		protocols = append(protocols, openrtb.Protocol(protocol))
```

Comment:
I don't think this does what you intend.

`make([]openrtb.Protocol, 5)` makes a slice with 5 elements (filled with whatever the "zero" is).
`make([]openrtb.Protocol, 0, 5)` makes a slice with 0 elements, but it promises not to re-allocate a new array object until you've added at least 6 things to it.

So if `unit.Video.protocols` has 5 elements, then I think this `protocols` array will have _10_ elements, after this block.